googler

```diff
@@ -2277,7 +2286,7 @@ class Result(object):
             else:
                 print(' %s%-*s %s %s' % (' ' * pre, indent, index + '.', title, url))
 
-    def _print_metadata_and_abstract(self, abstract, metadata=None, indent=5, pre=0):
+    def _print_metadata_and_abstract(self, abstract, metadata=None, keywords=[], indent=5, pre=0):
```

Comment:
Use `None` for the default value. Don't use a mutable default value for a keyword argument. See for instance https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments.

Reply:
(I know, we don't actually mutate this specific argument in the method body, but it's code smell.)